doc/api/esm.md

```diff
@@ -801,6 +801,22 @@ property:
 
 ## Differences Between ES Modules and CommonJS
 
+### Absolute file paths don't work on Windows
+
+`import` accepts _absolute specifiers_. These are similar to _absolute paths_
+on linux and MacOS, and therefore they are also accepted.
+But on Windows, importing an _absolute path_ to a file will not work.
+To ensure cross platform compatibility, file paths should be converted into
+URLs.
+
+```js
+import { pathToFileURL } from 'url';
+
+const fileUrl = pathToFileURL('c:\\path\\to\\file.js').href;
```

Comment:
```suggestion
const fileUrl = pathToFileURL('C:\\path\\to\\file.js').href;
```

uppercase is necessary. We are using `C:` everywhere else in those docs.

related PR: https://github.com/nodejs/node/pull/32294